tests/chibi-irregex.scm

```diff
@@ -0,0 +1,10 @@
+(import (chibi match)
+        (chibi test)
+        (scheme small)
+        (scheme r5rs)
+        (srfi 130))
+
+(load "irregex.scm")
```

Comment:
nit: We should use modules for implementations which support R7RS but that can be done as a follow-up.